applications/redis-app/src/main/java/com/bloxbean/cardano/yaci/store/redis/app/storage/CustomUtxoStorage.java

```diff
@@ -0,0 +1,72 @@
+package com.bloxbean.cardano.yaci.store.redis.app.storage;
+
+import com.bloxbean.cardano.yaci.store.common.domain.AddressUtxo;
+import com.bloxbean.cardano.yaci.store.common.domain.Amt;
+import com.bloxbean.cardano.yaci.store.core.domain.Cursor;
+import com.bloxbean.cardano.yaci.store.core.service.CursorService;
+import com.bloxbean.cardano.yaci.store.extensions.redis.utxo.impl.RedisUtxoStorage;
+import com.bloxbean.cardano.yaci.store.extensions.redis.utxo.impl.repository.RedisUtxoRepository;
+import com.bloxbean.cardano.yaci.store.utxo.storage.impl.UtxoCache;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+@Component
+public class CustomUtxoStorage extends RedisUtxoStorage {
```

Comment:
I think we should add some comments for this class, something like: https://github.com/bloxbean/metadata-indexer/blob/main/src/main/java/com/bloxbean/cardano/metdataindexer/CustomMetadataStorage.java